uint64 objectID
GPSSample gpsSample
uint32 width
uint32 length